content/blog/2024-07-21-RLadies-Bioconductor-collab/index-en.qmd

```diff
@@ -0,0 +1,50 @@
+---
+title: "Invitation to participate in the new R-Ladies–Bioconductor collaboration!"
+description: |
+  Announcing a new collaboration between R-Ladies and Bioconductor
+author:
+  - name: Janani Ravi
+    url: https://github.com/jananiravi
+date: 2024-07-21
+categories:
+  - RLadies
+  - Bioconductor
+image: "rladies-bioc.png"
+editor: 
+  markdown: 
+    wrap: sentence
+---
+
+![](rladies-bioc.png){ width=50% fig-align="center"}
+
+We are excited to invite you to join a new collaboration between two global communities, each committed to building diverse developer communities around the world, R-Ladies and Bioconductor.
+
+[R-Ladies](http://rladies.org) is a worldwide organization whose mission is to promote gender diversity in the R community.
+Starting in 2012 in San Francisco, USA, R-Ladies now has expanded to more than 200 chapters spanning 60+ countries around the world.
+R-Ladies supports endeavors with other like minded communities such as [LatinR](https://latin-r.com/en/), [RainbowR](https://rainbowr.netlify.app/), [miR](https://mircommunity.com/), [ArabR](https://arabr.github.io), [AfricaR](https://r4africa.org/), and [AsiaR](https://github.com/asiar-community).
+
+[Bioconductor](https://www.bioconductor.org/) develops, supports, and disseminates free open-source software that facilitates rigorous and reproducible analysis of data from current and emerging biological assays.
+The organization is dedicated to building a diverse, collaborative, and welcoming community of developers and data scientists.
+To this end, they have periodically elected scientific, technical, and community advisory boards committed to the technical and community aspects of Bioconductor.
+
+We are excited to begin a new initiative that aims to connect R-Ladies members who are also Bioconductor users or developers.
+This effort is designed as a key step towards bringing together passionate individuals committed to expanding diversity in the computational biology and bioinformatics developer space.
+
+Here are a few concrete ways in which we want to get the ball rolling:
+
+-   Announce each others' events on websites or social media
+-   Jointly host events
+-   Promote R-Ladies local watch parties of live-casted or pre-recorded events and professional networking events for Bioconductor conferences (BioC: BioC North America, BioCAsia, EuroBioC) Highlight Bioconductor speakers at R + computational biology R-Ladies events across chapters
+
+**Get Involved**
+
+Excited to join us in this partnership?
+Here's how you can get started:
+
+-   Learn more about our joint efforts at the [Bioconductor Working Group page](https://workinggroups.bioconductor.org/currently-active-working-groups-committees.html#r-ladies-and-bioc).
```

Comment:
This is a surprise, we didn't know about this working group.  As we said many times before, we don't allow chapters or groups to use our name inside the institution.

Reply:
Yani, this is just the working group that started the conversations with you -- a group of us from bioconductor (who are also R-Ladies) who are working towards a possible collaboration resulting in this initiative. This started after our initial call and shared interest -- we will change the language to rm partnership and add initiative/collaboration, instead (as suggested by Shannon).

Reply:
I've changed partnership to collaboration in the PR here: https://github.com/Bioconductor/BiocWorkingGroups/pull/45